src/components/learnMore/constants.js

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable max-len */
+/* eslint-disable sort-keys */
+
+/** @constant */
+export const LEARN_MORE_CLASSNAMES = {
+    HEADER: 'header',
+    IMAGE: 'image',
+    TEXT: 'text',
+    BODY: 'body',
+    ALT: 'image',
+    MORE: 'more',
+    ICON: 'icon',
+    FAST_DELIVERY_IMG: './images/raspberries-3757817.jpg',
+};
+
+export const TEXT1 =
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.';
```

Comment:
instead of defining all this fake text why dont you use faker nom module to auto generate them. Your PR is bloated before you have about 100 lines of lorem ipsum

Reply:
OK I will do that